components/automate-chef-io/content/docs/iam-v2-overview.md

```diff
@@ -10,7 +10,7 @@ toc = true
     weight = 10
 +++
 
-## Features in IAM v2
+## Features of IAM
```

Comment:
@kagarmoe We're going to have a merge conflict here. This header was already removed in 3033. :)